lib/sphinx-preview.js

```diff
@@ -36,13 +43,54 @@ export default {
     }
   },
 
-  getBindingPath() {
+  getCurrProjectPath() {
+    const editor = atom.workspace.getActivePaneItem();
+    const activeFilePath = editor.buffer.file.path;
+
+    if (!activeFilePath) return null;
+
+    for (const path of atom.project.getPaths()) {
+      if (activeFilePath.startsWith(path)) {
+        return path;
+      }
+    }
+    return null;
+  },
+
+  getContainerIdx(serverPath) {
+    if (!(serverPath in this.servers)) {
+      this.servers[serverPath] = this.idxs;
+      this.idxs += 1;
+    }
+
+    return this.servers[serverPath];
+  },
+
+  getContainerName(serverPath) {
+    var hash =  new Hashes.SHA256
+    var subName = hash.hex(serverPath);
+    var subName = subName.substring(0, 12);
+
+    return 'sphinx-preview-' + subName;
+  },
+
+  createContainerPort(success) {
+      FP(30000, (err, freePort) => {
+        return success(freePort)
+      })
+  },
+
+  getContainerPort(serverPath) {
+     return this.containerPorts[serverPath];
+  },
+
+  getBindingPath(projectPath) {
     // == DARWIN DOCKER FIX [1/2] === -- BEGIN --
     // Lines below were changed in order to fix filesystem-related issues
-    // preventing python-livereload's file watcher from running correctly 
+    // preventing python-livereload's file watcher from running correctly
     // when docker host is macOS (darwin) Mojave
 
-    let path = `${atom.project.getPaths()[0].replace(/[/\\*]/g, '/').replace(':', '')}`;
+    let path =  `/${projectPath.replace(/[/\\*]/g, '/').replace(':', '')}`; // WINDOWS
```

Comment:
```suggestion
    let path =  `${projectPath.replace(/[/\\*]/g, '/').replace(':', '')}`;
```

The leading slash for supporting windows is already added only when necessary in the lines below.
(In order to not have it when running on macOS)